src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,363 @@
+
+  &.pf-m-hover,
+  &:hover {
+    border-top: var(--pf-c-sequence__control--hover--BorderTopHeight) solid var(--pf-c-sequence__control--hover--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--hover--BorderBottomHeight) solid var(--pf-c-sequence__control--hover--BorderBottomColor);
+  }
+
+  &.pf-m-active,
+  &:active {
+    border-top: var(--pf-c-sequence__control--active--BorderTopHeight) solid var(--pf-c-sequence__control--active--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--active--BorderBottomHeight) solid var(--pf-c-sequence__control--active--BorderBottomColor);
+  }
+
+  &.pf-m-focus,
+  &:focus {
+    border-top: var(--pf-c-sequence__control--focus--BorderTopHeight) solid var(--pf-c-sequence__control--focus--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--focus--BorderBottomHeight) solid var(--pf-c-sequence__control--focus--BorderBottomColor);
+  }
+}
+
+.pf-c-sequence__control-left {
+  left: var(--pf-c-sequence__control-left--PositionLeft);
+  background-color: var(--pf-c-sequence__control-left--BackgroundColor);
+  border-right: var(--pf-c-sequence__control-left--BorderRightWidth) solid var(--pf-c-sequence__control-left--BorderRightColor);
+}
+
+.pf-c-sequence__control-right {
+  right: var(--pf-c-sequence__control-right--PositionRight);
+  background-color: var(--pf-c-sequence__control-right--BackgroundColor);
+  border-left: var(--pf-c-sequence__control-right--BorderLeftWidth) solid var(--pf-c-sequence__control-right--BorderLeftColor);
+}
+
+.pf-c-sequence__control-icon {
+  color: var(--pf-c-sequence__control-icon--Color);
+}
+
+// Horizontal list
+.pf-c-sequence__list {
+  display: inline-flex;
+  overflow: scroll;
+}
+
+// Change default styles for the list and list items
+.pf-c-sequence__list .pf-c-sequence__item {
+  list-style-type: none;
+
+  // added so that the list will be read by screen reader
+  // &::before {
+  //   content: "/200B";
+  // }
+}
+
+// Items inside of list
+.pf-c-sequence__item {
+  max-width: var(--pf-c-sequence__item--MaxWidth);
+  margin-right: var(--pf-c-sequence__item--MarginRight);
+  margin-left: var(--pf-c-sequence__item--MarginLeft);
+}
+
+// Vertically centers number and border
+.pf-c-sequence__number-container {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+// Sequence Link Position Number
+.pf-c-sequence__number {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-width: var(--pf-c-sequence__number--MinWidth);
+  max-height: var(--pf-c-sequence__number--MaxHeight);
+  padding: var(--pf-c-sequence__number--Padding);
+  margin-right: var(--pf-c-sequence__number--MarginRight);
+  font-size: var(--pf-c-sequence__number--FontSize);
+  color: var(--pf-c-sequence__number--Color);
+  text-align: center;
+
+  &::before {
+    position: absolute;
```

Comment:
This is a great question! We should discuss this at our a11y meeting since this probably mostly affects screen reader users and keyboard-only users.

Reply:
@jgiardino sounds good! and we can always leave the source order as is if needed, still use non-absolute positioning, and use flex `order` to re-arrange visually.

Reply:
Taken absolute position off of the controls and they are placed within the sequence using flex order to re-arrange them visually.

Reply:
I think this [codepen](https://codepen.io/benfrain/full/zZZLaP/) was the source of inspiration for this design. When I tested this example using only a keyboard and also using voiceover, the arrow buttons disappear before I would be able to place focus on them, because in both cases, the tabs scroll to the end as I shift focus. 

Because of this, I don't think DOM order is that important, since they're not needed for keyboard-only navigation. 

But we still want to keep the `aria-label` attributes as you have them for screen reader users who are using a mouse or touch.

Reply:
Also can we set a min-width on these buttons so that the target area abides by the 44x44px minimum? the var for that is `--pf-global--target-size--MinWidth`

Reply:
yep, just updated that